dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -297,5 +297,9 @@ LOGOUT_TOKEN_SID_REQUIRED_BUT_MISSING=CWWKS1551E: The logout token is missing th
 LOGOUT_TOKEN_SID_REQUIRED_BUT_MISSING.explanation=The OpenID Connect client requires that the logout token contains a session ID to identify the OpenID Connect client session with the OpenID Connect provider.
 LOGOUT_TOKEN_SID_REQUIRED_BUT_MISSING.useraction=Set the backchannelLogoutSessionRequired OpenID Connect client configuration attribute to false, or ensure that the logout token contains a "sid" claim.
 
+BACKCHANNEL_LOGOUT_PERFORM_LOGOUT_FAILED=CWWKS1552E: The OpenID Connect client encountered the following error while performing back-channel logout with a logout token with claims {0}: {1}
```

Comment:
Change "the following" to "an" 

(nit, not a required change)

Reply:
the nls is no longer needed.
i've removed it from the file.